src/containers/Application.js

```diff
@@ -34,16 +35,19 @@ class Application extends Component {
 		this.HeaderService = new HeaderService(this, "HeaderService");
 		this.ReduxService = new ReduxService(this, "ReduxService");
 
+				
+		
 		// Instantiate modules
 		for (var i in props.modules) {
 			const module = new props.modules[i](this);
 			this.Modules.push(module);
 		}
 
-		// Create store
+		// Create store		
 		this.Store = Object.keys(this.ReduxService.Reducers).length > 0
 					? createStore(combineReducers(this.ReduxService.Reducers))
-					: createStore((state) => state)
+					: createStore((state) => state)		
```

Comment:
Pls removoe those `		` spaces at the end of lines. You might want to configure your editor or something